modules/distribution/carbon-home/resources/dashboards/rdbms-statistics/WSO2 Streaming Integration - RDBMS Table Statistics.json

```diff
@@ -16,14 +16,14 @@
   "gnetId": null,
   "graphTooltip": 0,
   "id": 13,
-  "iteration": 1590759359856,
+  "iteration": 1592292338342,
   "links": [
     {
       "icon": "dashboard",
       "tags": [],
-      "title": "WSO2 Streaming Integrator - RDBMS Statistics",
+      "title": "WSO2 Streaming Integration - RDBMS Statistics",
```

Comment:
Why this is changed to Streaming Integration since the product is Streaming Integrator. Check other places as well

Reply:
Resolve in 094705475651d37ee5e03f75532cad40a82e7d82